packages/@office-iss/react-native-win32/src/Libraries/Components/ProgressViewIOS/ProgressViewIOS.win32.js

```diff
@@ -1,8 +1,45 @@
 /**
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
  * @format
  */
+
 'use strict';
+import * as React from 'react';
+import StyleSheet from '../../StyleSheet/StyleSheet';
+import Text from '../../Text/Text';
+import View from '../View/View';
+
+class DummyProgressViewIOS extends React.Component {
+  render() {
+    return (
+      <View style={[styles.dummy, this.props.style]}>
+        <Text style={styles.text}>
+          ProgressViewIOS is not supported on this platform!
+        </Text>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  dummy: {
+    width: 120,
+    height: 20,
+    backgroundColor: '#ffbcbc',
+    borderWidth: 1,
+    borderColor: 'red',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  text: {
+    color: '#333333',
+    margin: 5,
+    fontSize: 10,
+  },
+});
 
-module.exports = require('../UnimplementedViews/UnimplementedView');
+module.exports = DummyProgressViewIOS;
```

Comment:
Just for my knowledge, can you elaborate on this one:
> Move UnimplementedView stubs to the upstream impl to avoid future conflicts

Reply:
We had a conflict due to the require->import conversion - what more can happen here?

Reply:
Upstream there was a bulk conversion from require to import. For some of these view stubs we had our own code that just exported UnimplementedView, instead of using the upstream copy which is a bit more specialized. The exact upstream copy works just fine, and now with merge tooling, matching upstream means less conflicts in the future.